docs/API.md

```diff
@@ -473,7 +473,7 @@ main = do
 ```
 
 <a name="fGetObject"></a>
-### fGetObject :: Bucket -> Object -> FilePath -> Minio ()
+### fGetObject :: Bucket -> Object -> FilePath -> GetObjectOptions -> Minio ()
```

Comment:
We would also need to update the doc for `getObject`. However, it should be ok to update it in a subsequent change.

Reply:
I didn't add since putObject() is not added as well. So i thought we didn't have to add them.